src/pages/about.html

```diff
@@ -0,0 +1,28 @@
+<<!DOCTYPE html>
```

Comment:
Issue caused by typo. 
Should be `<!DOCTYPE HTML>`, you have two left arrows `<<`.